gradoop-flink/src/main/java/org/gradoop/flink/io/impl/csv/functions/ElementToCSV.java

```diff
@@ -115,6 +116,19 @@ String collectionToCsvString(Collection<?> collection) {
       .collect(Collectors.joining(CSVConstants.LIST_DELIMITER, "[", "]"));
   }
 
+  /**
+   * Returns a CSV string representation of the temporal attributes in format:
+   *
+   * {@code (tx-from,tx-to),(val-from,val-to)}
+   *
+   * @param transactionTime the transaction time
+   * @param validTime the valid time
+   * @return CSV string representation
+   */
+  String getTemporalDataString(Tuple2<Long, Long> transactionTime, Tuple2<Long, Long> validTime) {
+    return transactionTime + CSVConstants.LIST_DELIMITER + validTime;
```

Comment:
This relies on implementation details of Flink, the format of `Tuple::toString` is (afaik) not defined, so format of the return value can not be guaranteed to be correct. Maybe use `String.format` without an implicit `Tuple::toString` instead.